locales/fr_FR/pcsx2_Main.po

```diff
@@ -2668,7 +2651,7 @@ msgstr "Cette langue deviendra celle par défaut !"
 
 #: pcsx2/gui/Panels/PathsPanel.cpp:38
 msgid "Cheats:"
-msgstr ""
+msgstr "Codes triches:"
```

Comment:
Les codes triches

Reply:
Les codes triches sounds better for me. Fixing this.

Reply:
First rule of french i learned Les must be used and means The.
Otherwise use Des which is silent. You can always correct me.